lib/src/dio-interceptor-wrapper.dart

```diff
@@ -62,6 +64,15 @@ class SuperTokensInterceptorWrapper extends Interceptor {
       options.headers[antiCSRFHeaderKey] = antiCSRFToken;
     }
 
+    SuperTokensTokenTransferMethod tokenTransferMethod =
+        SuperTokens.config.tokenTransferMethod!;
+    options.headers["st-auth-mode"] =
+        tokenTransferMethod == SuperTokensTokenTransferMethod.COOKIE
+            ? 'cookie'
+            : 'header';
+
+    options = await _setAuthorizationHeaderIfRequired(options);
+
```

Comment:
Need to check for duplicate cookies when combining user defined cookies and those from the cookie manager